pep-0458.txt

```diff
@@ -828,6 +828,14 @@ version of the *snapshot* metadata, which in turn lists the versions of the
 *targets* and delegated targets metadata, all as part of a given consistent
 snapshot.
 
+In normal usage, version number overflow is unlikely to occur as an 8-byte
+integer can be incremented once per millisecond and last almost 300 million
+years. If an attacker increases the version number arbitrarily
+(e.g. to MAX_INT), the repository can recover by revoking the snapshot role
```

Comment:
I wonder if it is clear to the reader what MAX_INT is. Also, not only snapshot role may need to be revoked, timestamp and top-level targets may too. Maybe we can say: 

*"If an attacker increases the version number arbitrarily, the repository can recover by revoking the compromised keys and reset the version number as described in the TUF specification__."*